src/mw-layout/directives/mw_header.js

```diff
@@ -8,14 +8,15 @@ angular.module('mwUI.Layout')
         url: '@?',
         mwTitleIcon: '@?',
         showBackButton: '=?',
-        mwBreadCrumbs: '=?'
+        mwBreadCrumbs: '=?',
+        description: '@?',
       },
       require: '^?mwUi',
       templateUrl: 'uikit/mw-layout/directives/templates/mw_header.html',
       link: function (scope, el, attrs, mwUiCtrl, $transclude) {
         $rootScope.siteTitleDetails = scope.title;
         BrowserTitleHandler.setTitle(scope.title);
-
+        scope.descriptionMargin = scope.mwTitleIcon ? '20px' : '0px';
```

Comment:
Why not via css? You can add a class like `description-open` when toggeling description  and set the margin when this class is set